### PR TITLE
Add print for flaky test to make debug easier next time it fails

### PR DESCRIPTION
**Pull request by burqen**

### CODE REVIEW

**Comment by burqen** [comment]
@MishaDemianenko Care to take a look?

**Comment by MishaDemianenko** [comment]
@burqen lets re-target it to 3.2 to avoid burden with merging/approving etc.

File: community/server/src/test/java/org/neo4j/server/rest/transactional/integration/TransactionIT.java [hunk 1]
```diff
@@ -272,6 +272,13 @@ public void begin_and_execute_periodic_commit_and_commit() throws Exception
             );
 
             long txIdAfter = resolveDependency( TransactionIdStore.class ).getLastClosedTransactionId();
+
+            // todo This test is flaky and hard to reproduce.
+            // todo Printing response for easier debug next time it fails
+            // todo System.out is suppressed and will only be printed in case of test failure
+            System.out.println( "RESPONSE:" );
+            System.out.println( response );
```
Comment (MishaDemianenko) [inline community/server/src/test/java/org/neo4j/server/rest/transactional/integration/TransactionIT.java:280]: Are you sure that this response have useful toString?

Comment (burqen) [reply community/server/src/test/java/org/neo4j/server/rest/transactional/integration/TransactionIT.java:280]: Yes

**Pull request merged**